### ISSUE TITLE
Adjust surveillance banner on advocacy site after the vote

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/movement-building#53]
After Congress votes on the surveillance issue, the text and CTA on the surveillance banner will need to change. Can we draft some text (version a: celebratory, version b: sadface but resolute). I assume a blog will be written either way, and the button can change to 'read more' and link to the blog.

cc @valianttry

**Comment by edrushka** [comment]
@DaveSteer can you briefly outline what the outcomes of the vote might be? I want to be prepared with copy for the different scenarios. Thanks.

**Comment by Jochai1** [comment]
Hey @edrushka I'm working on this messaging right now. But here are the four possible scenarios:

1) USA FREEDOM passes (end result: reform!) <-- this is what we want
2) USA FREEDOM fails (end result: sunset)
3) A two month reauthorization passes the Senate and the House is recalled and passes a sister reauthorization (end result: delay leading to weakened or no reform)
4) A two month reauthorization passes the Senate and the House stays on recess (end result: sunset)

**Comment by edrushka** [comment]
Thanks, @jochai1 This is super helpful.

**Comment by edrushka** [comment]
Update: Senate voted last Friday. [More info here.](http://www.theverge.com/2015/5/24/8652679/nsa-preparing-to-shut-down-bulk-surveillance-programs)

Once we confirm our messaging, we'll need to update the advocacy page. cc @valianttry @DaveSteer @Jochai1 @simonwex

**Comment by valianttry** [comment]
:clap: Obama signed the bill into law! We're winners!! :+1: 

@KevZawacki has our official comms on this, and he knows if there's a blog, and you can find our email draft here for any additional language that might be helpful:
https://docs.google.com/document/d/1YisaSfroR_auA_YnwAw7P5DIuWKPICSxL_o_Bun9KSw/edit

**Comment by KevZawacki** [comment]
Hey! To get the ball rolling on banner copy:

Hed: A Big Victory for Privacy and the Open Web

Copy: Together, we stood up and demanded Congress reform mass surveillance programs. And we made a difference: this June, the USA FREEDOM Act was signed into law, a significant step toward restoring trust online.  

Button: Read more [https://blog.mozilla.org/netpolicy/2015/06/02/mozilla-applauds-u-s-senates-passage-of-the-usa-freedom-act/]

**Comment by DaveSteer** [comment]
Thanks for getting the ball rolling, @KevZawacki 
The copy seems long for a banner. Can we shorten?

--Dave

**Comment by KevZawacki** [comment]
For sure. How about:

Hed: A Big Victory for Privacy

Copy: Together, we demanded Congress reform mass surveillance. And we made a difference: the USA FREEDOM Act is a major step toward restoring trust online. 

Button: Read more [https://blog.mozilla.org/netpolicy/2015/06/02/mozilla-applauds-u-s-senates-passage-of-the-usa-freedom-act/]

**Comment by edrushka** [comment]
@DaveSteer @Jochai1 are you good with this copy? We should _really_ try to update this before we point people to the site for the OWF announcment.

**Comment by DaveSteer** [comment]
Works for me. Thanks!

**Comment by edrushka** [comment]
@alicoding are you able to change the surveillance banner to:
- [x] Headline: A Big Victory for Privacy
- [x] Body: Together, we demanded Congress reform mass surveillance. And we made a difference: the USA FREEDOM Act is a major step toward restoring trust online. 
- [x] Button text: Read more
- [x] Button link: https://blog.mozilla.org/netpolicy/2015/06/02/mozilla-applauds-u-s-senates-passage-of-the-usa-freedom-act/

IMPORTANT: This should be rolled into Monday's update. cc @jbuck
